alvenir/guidance/v1/guidance.proto

```diff
@@ -1,67 +1,73 @@
 syntax = "proto3";
+
 package alvenir.guidance.v1;
 
 
 message LlmPromptAnswer {
-  string prompt = 1;
-  string context = 2;
-  string answer = 3;
+    string prompt = 1;
+    string context = 2;
+    string answer = 3;
 }
 
 message LlmData {
-  LlmPromptAnswer introduction = 1;
-  LlmPromptAnswer client_understanding = 2;
-  LlmPromptAnswer follow_up_questions = 3;
-  LlmPromptAnswer answer_quality = 4;
-  LlmPromptAnswer language = 5;
-  LlmPromptAnswer conclusion = 6;
-  LlmPromptAnswer follow_up = 7;
-  LlmPromptAnswer general_feedback = 8;
+    LlmPromptAnswer introduction = 1;
+    LlmPromptAnswer client_understanding = 2;
+    LlmPromptAnswer follow_up_questions = 3;
+    LlmPromptAnswer answer_quality = 4;
+    LlmPromptAnswer language = 5;
+    LlmPromptAnswer conclusion = 6;
+    LlmPromptAnswer follow_up = 7;
+    LlmPromptAnswer general_feedback = 8;
 }
 
 message CallMeta {
-  string time_of_call = 1;
-  string length = 2;
-  float answer_time = 3;
+    string time_of_call = 1;
+    string length = 2;
+    float answer_time = 3;
 }
 
 message TranscriptionMeta {
-  string n_words = 1;
-  string n_fill_words = 2;
+    string n_words = 1;
+    string n_fill_words = 2;
 }
 
 // for timeline chart
-message StringData {
-  repeated string x = 1;
-  repeated string y = 2;
-  string speaker = 3;
+message TimeInterval {
+    string start = 1;
+    string end = 2;
+}
+
+message TimeLineData {
+    string speaker = 1;
+    repeated TimeInterval interval_data = 2;
 }
 
-// for any chart with numeric values
-message FloatData {
-  repeated float x = 1;
-  repeated float y = 2;
-  string speaker = 3;
+message TimeLineConcat {
+    TimeLineData caller_data = 1;
+    TimeLineData advisor_data = 2;
 }
 
+// for sentiment chart
+message SentimentCoordinates {
+    float x = 1;
+    float y = 2;
+}
 
-message StringDataConcat {
-  string title = 1;
-  StringData caller_data = 2;
-  StringData advisor_data = 3;
+message SentimentData{
+    string speaker = 1;
+    repeated SentimentCoordinates data_points = 2;
 }
 
-message FloatDataConcat {
-  string title = 1;
-  FloatData caller_data = 2;
-  FloatData advisor_data = 3;
+message SentimentConcat {
```

Comment:
What is a concat? :)

Reply:
concatenation of advisor and caller data :upside_down_face: